spec.html

```diff
@@ -35052,6 +35052,18 @@
               1. Let _onRejected_ be *undefined*.
             1. Let _fulfillReaction_ be the PromiseReaction { [[Capabilities]]: _resultCapability_, [[Type]]: `"Fulfill"`, [[Handler]]: _onFulfilled_ }.
             1. Let _rejectReaction_ be the PromiseReaction { [[Capabilities]]: _resultCapability_, [[Type]]: `"Reject"`, [[Handler]]: _onRejected_ }.
+            1. Return EnqueuePromiseReactions(_promise_, _fulfillReaction_, _rejectReaction_, _resultCapability_).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-enqueuepromisereactions" aoid="EnqueuePromiseReactions">
+          <h1>EnqueuePromiseReactions ( _promise_, _fulfillReaction_, _rejectReaction_, _resultCapability_ )</h1>
+          <p>The abstract operation EnqueuePromiseReactions enqueues PromiseJobs with the provided PromiseReaction records on _promise_. The result is _resultCapability_'s promise.</p>
+          <emu-alg>
+            1. Assert: IsPromise(_promise_) is *true*.
```

Comment:
They're definitely not *necessary*; nothing will change depending on their presence or absence. The question is whether they're helpful or annoying/distracting, and I claim the latter. In general abstract ops don't do type assertions on their arguments, except in cases where it could be potentially unclear (e.g. what type of completion record they will end up getting). This has only one call site so it seems super-unnecessary.